sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -229,7 +230,7 @@ options.Diagnostics.LoggedQueryParameters.Remove("api-version");
 ```xml
  <ItemGroup>
     <RuntimeHostConfigurationOption Include="Azure.Core.Pipeline.DisableHttpWebRequestTransport" Value="true" />
-  </ItemGroup> 
+  </ItemGroup>
```

Comment:
nit: have we made sure IDE settings across the team aren't going to cause whitespace edit wars across changes?